### PR TITLE
Page to create new users

### PR DESCRIPTION
**Pull request by Ygoraphael**

### CODE REVIEW

**Review by dariia-kachalova — COMMENTED** [review]
Good

**Review by dariia-kachalova — COMMENTED** [review]
+

**Review by dangoth — APPROVED** [review]

**Review by VladyslavCherednychenko — APPROVED** [review]

**Review by dariia-kachalova — APPROVED** [review]

File: react-portal-employees/src/components/Employee/createEmployee.tsx [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+import Box from '@mui/material/Box';
+
+import ApiEmployee from '../../services/api/TasksEmployee/ApiEmployee';
+
+const theme = createTheme();
+
+export default function CreateEmployee() {
+    const [input, setInput] = useState({
+      username: '',
+      EmailAddress: '',
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/Employee/createEmployee.tsx:19]: Why case is inconsistent?

File: react-portal-employees/src/components/Employee/createEmployee.tsx [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+      confirmPassword: '',
+      email: '',
+      FirstName: '',
+      LastName: '',
+      DateOfBirth: ''
+    })   
+
+    const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+      let MsgEr = "";
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/Employee/createEmployee.tsx:50]: use camelCase for front-end pls

File: react-portal-employees/src/components/Employee/createEmployee.tsx [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+      Object.entries(error).forEach(
+        ([key, value]) => {
+          if (value != "") MsgEr = value;
+        }
+      );
+      if (MsgEr != ""){
+        alert(MsgEr);
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/Employee/createEmployee.tsx:57]: let's not use alert, use some toast notification instead

File: react-portal-employees/src/components/Employee/createEmployee.tsx [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+      ApiEmployee.CreateEmployee(JSON.stringify(input));
+    };
+
+    const checkValidation = (e: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement | HTMLSelectElement>) => {
+      const { name, value } = e.target;
+      if (name == "Seniority" || name == "Role")
+      {
+        setInput(prev => ({
+          ...prev,
+          [name]: Number(value)
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/Employee/createEmployee.tsx]: why extra convert to number here?

File: react-portal-employees/src/components/Employee/createEmployee.tsx [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+        }));
+      }
+      else if (name == "confirmPassword") { setConfirmPwd(value); }
+      else{
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/Employee/createEmployee.tsx:66]: format file pls, use our prettier settings to highlight the formatting issues

File: react-portal-employees/src/components/Employee/createEmployee.tsx [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+                autoComplete="DateOfBirth"
+                type="Date"
+                onChange={e => checkValidation(e)}
+                error={!!error?.DateOfBirth}
+                helperText={error.DateOfBirth}
+              />
+              <InputLabel htmlFor="Seniority">Seniority</InputLabel>
+              <NativeSelect id="Seniority" name="Seniority" fullWidth required autoFocus onChange={e => checkValidation(e)}>
+                <option value=""></option>
+                <option value="0">Junior</option>
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/Employee/createEmployee.tsx]: introduce enum for that pls and go through it

File: react-portal-employees/src/components/Employee/createEmployee.tsx [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+                <option value="1">Mid Level</option>
+                <option value="2">Senior</option>
+              </NativeSelect>
+              <InputLabel htmlFor="Role">Role</InputLabel>
+              <NativeSelect id="Role" name="Role" fullWidth required autoFocus onChange={e => checkValidation(e)}>
+                <option value=""></option>
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/Employee/createEmployee.tsx:204]: same

**Review by DmytroDorodnykhTeamInternational — APPROVED** [review]
Please, add a screenshot of how UI looks like for EACH front-end pr

**Comment by DmytroDorodnykhTeamInternational** [comment]
Please, add a screenshot of how UI looks like for EACH front-end pr

File: react-portal-employees/src/components/Employee/createEmployee.tsx [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+      if (MsgEr != ""){
+        alert(MsgEr);
+        return
+      } 
+      ApiEmployee.CreateEmployee(JSON.stringify(input));
+    };
+
+    const checkValidation = (e: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement | HTMLSelectElement>) => {
+      const { name, value } = e.target;
+      if (name == "Seniority" || name == "Role")
```
Comment (Tsarar) [inline react-portal-employees/src/components/Employee/createEmployee.tsx]: "Seniority" should be moved to a constant string with the other similar names

**Comment by Ygoraphael** [comment]
How page create employee looks like
image

**Comment by Ygoraphael** [comment]
How it looks like when have erros
image

**Pull request merged**